plugins/controlboard/src/ControlBoard.cpp

```diff
@@ -97,27 +102,21 @@ void ControlBoard::Configure(const Entity& _entity,
         return;
     }
 
-    std::string deviceName = m_pluginParameters.find("yarpDeviceName").asString();
+    std::string yarpDeviceName = m_pluginParameters.find("yarpDeviceName").asString();
 
     m_robotScopedName = gz::sim::scopedName(_entity, _ecm, "/");
     yDebug() << "gz-sim-yarp-controlboard-system : robot scoped name: " << m_robotScopedName;
-
-    m_deviceScopedName
-        = m_robotScopedName + "/" + m_pluginParameters.find("yarpDeviceName").asString();
-    yDebug() << "gz-sim-yarp-controlboard-system : device scoped name: " << m_deviceScopedName;
+    yDebug() << "gz-sim-yarp-controlboard-system : yarpDeviceName: " << yarpDeviceName;
 
     m_modelEntity = _entity;
 
-    m_controlBoardData.controlBoardId = m_deviceScopedName;
+    // m_controlBoardData.controlBoardId = m_deviceId;
 
     m_pluginParameters.put(yarp::dev::gzyarp::YarpControlBoardScopedName.c_str(),
                            m_robotScopedName.c_str());
 
-    // Insert the pointer in the singleton handler for retrieving it in the yarp driver
-    ControlBoardDataSingleton::getControlBoardHandler()->setControlBoardData(&(m_controlBoardData));
-
     m_pluginParameters.put("device", "gazebo_controlboard");
-    m_pluginParameters.put("controlBoardId", m_deviceScopedName);
+    // m_pluginParameters.put("controlBoardId", m_deviceId);
```

Comment:
Can we remove this?

Reply:
Solved in https://github.com/robotology/gz-sim-yarp-plugins/pull/153/commits/fd8043fb3c1ec219c31f9d35cd441383ffcfa57b